src/ui/VideoArea.qml

```diff
@@ -559,24 +559,28 @@ Item {
         return false;
     }
     function detectVideoSequence(folder: url, filename: string): var {
-        // url pattern, new path function, additional condition
+        // url pattern, 1st file index, new path function
         const patterns = [
-            // GoPro
-            [/(G[XH](01).+\.MP4)$/i, function(match, i) {
+            // GoPro 1-5
+            [/((?:GOPR|GP\d{2})(\d{4})\.MP4)$/i, 0, function(match, i) {

```

Comment:
are you sure this is correct? It captures `GOPR` without a number or `GP` with 2 digits, so what's the point of capturing GOPR\d\d\d\d, since we don't know which part is it, but only the 4-digit clip number?

Reply:
Yes. It's the old naming convention of earlier versions (see the linked page), which is quite annoying to deal with.. and probably the reason they changed it to something that makes slightly more sense.

GOPR[clip] is always the first of the sequence, which in this scheme is set as index 0. The second part is GP01, and so on..